src/screens/login/loginScreen.jsx

```diff
@@ -33,6 +33,8 @@ class LoginScreen extends Component {
    * @param  {object}    props.actions holds the actions for this state
    * @param  {object}    props.navigation the navigation object provided by 'react-navigation'
    * @param  {Function}  props.scanSuccess function that is triggered when the qr-scanner picks something up
+   * @param  {boolean}   props.loginUnauthorized the navigation object provided by 'react-navigation'
+   * @param  {object}    props.loginError the navigation object provided by 'react-navigation'
```

Comment:
This looks like a copy-paste error to me.

Reply:
Definitely 🙈